pkg/server/smtp/listener.go

```diff
@@ -63,6 +64,7 @@ type Server struct {
 	manager        message.Manager    // Used to deliver messages.
 	listener       net.Listener       // Incoming network connections.
 	wg             *sync.WaitGroup    // Waitgroup tracks individual sessions.
+	TLSconfig      *tls.Config
```

Comment:
I'd rather not export this as it's not used outside of the smtp package:  `tlsConfig`